build-logic/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.13.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.1'
+}
+
+gradlePlugin {
+    plugins {
+        androidVersionsPlugin {
+            id = 'org.gradle.android.plugin'
```

Comment:
This doesn't seem like a very descriptive plugin id.  I'd make this `org.gradle.android-versions` or something like that.

Reply:
Agreed.